app/Http/Controllers/Users/MultiplayerController.php

```diff
@@ -19,39 +20,52 @@ public function index()
     {
         $request = request();
         $user = FindForProfilePage::find($request->route('user'));
```

Comment:
not really related but just noticed this can be just `$userId` if the function signature is `index($userId)`...